Classes/Service/SolrServiceProvider.php

```diff
@@ -1147,4 +1148,16 @@ protected function tagForFacet(string $facetID): string
     {
         return 'facet-'.$facetID;
     }
+
+    /*
+     * Set configured main query operator. Defaults to 'AND'.
+     */
+    private function addMainQueryOperator()
+    {
+        $mainQueryOperator = 'AND';
```

Comment:
This introduces a previously only implicit set operator `AND`. Why not leave this untouched and only do something in the `if`-clause below?